coding/src/lib.rs

```diff
@@ -399,4 +332,9 @@ mod test {
     fn test_suite_no_coding() {
         test_suite::<NoCoding<Sha256>>();
     }
+
+    #[test]
```

Comment:
I wonder how hard it would be to compare an off-the-shelf KZG impl here? Seems like that would be unreal...